docs/source/docker/advanced/active_learning.rst

```diff
@@ -1,7 +1,294 @@
 .. _ref-docker-active-learning:
 
-Active learning
-==============================================
+Active Learning
+===============
+
+Lightly makes use of active learning scores to select the samples which will yield
+the biggest improvements of your machine learning model. The scores are calculated
+on-the-fly based on model predictions and provide the selection algorithm with feedback
+about the uncertainty of the model for the given sample. 
+
+Learn more about the concept of active learning scores:
+:ref:`lightly-active-learning-scorers`.
+
+Prerequisites
+--------------
+In order to do active learning with Lightly, you will need the following things:
+
+- The installed Lightly docker (see :ref:`ref-docker-setup`)
+- A dataset with a configured datasource (see :ref:`ref-docker-with-datasource-datapool`)
+
+.. note::
+
+    The dataset does not need to be empty! For example, an initial selection without
+    active learning can be used to train a model. The predictions from this model
+    can then be used to improve your dataset by adding new images to it through active learning.
+
+
+Next, you need to prepare the model predictions in the correct format.
+
+Predictions
+-----------
+
+In the following, we will outline the format of the predictions required by the 
+Lightly docker. Everything regarding predictions will take place in a subdirectory
+of your configured datasource called `.lightly/predictions`.
+
```

Comment:
I would give a summary here. I got lost here as we just go deeper and deeper with the concepts in the following blocks and I don't know why. Maybe we summarize the rough folder structure we will have in the end for the predictions and why we have them that way before going into detail?